vue-components/src/components/InputWithExtender.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+	<div
+		:class="[ 'wikit', 'wikit-InputWithExtender' ]"
+	>
+		<label class="wikit-InputWithExtender__label" :for="id">{{ label }}</label>
```

Comment:
This is going to need the suffix slot as well, like the ones added in #386.

(We probably should create Label component or something like that at some point :thinking:)

Reply:
Done

Reply:
Almost, this needs a [wrapping `<span>`](https://github.com/wmde/wikit/pull/386/files#diff-2ea7a99f61da845846aec597ca9e2163ad2b2b0bf9e46b769e93860331d250aaR6-R9) and [some styles](https://github.com/wmde/wikit/pull/386/files#diff-2ea7a99f61da845846aec597ca9e2163ad2b2b0bf9e46b769e93860331d250aaR257-R261) so that it works nicely with rtl as well.